src/test/java/org/mule/test/http/functional/listener/HttpListenerDisableTimeoutsConfigTestCase.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.test.http.functional.listener;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.mule.runtime.core.api.Event;
+import org.mule.tck.junit4.rule.DynamicPort;
+import org.mule.tck.junit4.rule.SystemProperty;
+import org.mule.test.http.functional.AbstractHttpTestCase;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.mule.runtime.core.api.config.MuleProperties.SYSTEM_PROPERTY_PREFIX;
+
+public class HttpListenerDisableTimeoutsConfigTestCase extends AbstractHttpTestCase {
+
+  @Rule
+  public DynamicPort port = new DynamicPort("port");
+
+  @Rule
+  public SystemProperty disableTimeouts = new SystemProperty(SYSTEM_PROPERTY_PREFIX + "timeout.disable", "true");
```

Comment:
There should be a constant somewhere for "timeout.disable" and it must be referenced here